arrow/factory.py

```diff
@@ -64,14 +65,14 @@ def get(self, *args, **kwargs):
             <Arrow [2013-05-08T05:54:34.293378+00:00]>
 
             >>> arrow.get('1367992474')
-            <Arrow [2013-05-08T05:54:34+0struct_time0:00]>
+            <Arrow [2013-05-08T05:54:34+00:00]>
 
         **One** ISO-8601-formatted ``str``, to parse it::
 
             >>> arrow.get('2013-09-29T01:26:43.830580')
             <Arrow [2013-09-29T01:26:43.830580+00:00]>
 
-        **One** ``tzinfo``, to get the current time in that timezone::
+        **One** ``tzinfo``, to get the current time **converted** to that timezone::
```

Comment:
Here, the timezone is a conversion, but it is a replacement everywhere else in `get`.  I find this confusing and wonder if this particular case should be deprecated in favour of just calling `arrow.now(tz)` directly.